httpie/cli/definition.py

```diff
@@ -309,6 +309,21 @@
     '''
 )
 
+output_processing.add_argument(
+    '--response-content-type',
+    metavar='CONTENT_TYPE',
+    help='''
+        Overwrite the response Content-Type. Useful when the Content-Type returned
+        by the server is incorrect. It is a shortcut for:
```

Comment:
Maybe "Useful when the Content-Type returned by the server is incorrect." should be removed, WDYT?

Reply:
I think mentioning the main use case is a good idea. I’d also explain that it’s for display/formatting purposes only. But let’s look at the overall documentation together, I have some more ideas and we need to coordinate the `--help` output with README.